alf/trainers/policy_trainer.py

```diff
@@ -230,8 +238,9 @@ def train(self):
                 ps.print_callees()
 
                 logging.info(s.getvalue())
-        finally:
```

Comment:
Why making this change? Currently it will save the checkpoint if an error occurs, which is good.